header_for_5_lab.h

```diff
@@ -46,17 +46,17 @@ class FlowerShop {
             }
         }
     }
-     void displayTop() {
-        sortFlow();
-        cout << "TOP Flowers:" << '\n'
-            << assortment[0].name << " - " << assortment[0].price << '\n'
-            << assortment[1].name << " - " << assortment[1].price << '\n'
-            << assortment[2].name << " - " << assortment[2].price << endl;
+     void DisplayTop() {
```

Comment:
now you display all
maybe, try to pass to the method exactly the quantity that needs to be output